### PR TITLE
Convert Vision Map About Widget Content to HTML

### LINKED ISSUE #69: Vision page info to HTML
**Summary:**
Utilize images and fonts found [here](https://drive.google.com/drive/u/0/folders/1zwEmc9pz5FDueEOZBNiBQgEVG-jR1V5_) to re-create the [Vision page's](https://wfrc.org/wasatch-choice-map/#mapList=vision.transportation.landuse.econdev.recreation.ato&selectedMap=vision&sideBarClosed=false) informational panel as images and html (and css?) to match closely the existing style and layout.  
> On our Vision tab we've got a number of pngs for the vision text. We need to convert these to formatted html text...

> We are not going for a perfect match (for example, QR codes and weird 4 square bullets under partners can be dropped) but want it to stay true to overall look and feel.

**Outcome:**
HTML, image files and perhaps some css within the Vision tab's About config.

### PR DESCRIPTION
**Pull request by stdavis**
Also includes some upgrades/refactors to the About Widget:

* Adds a page for previewing all of the about widget content in one place: `/about-tests`
* Moves the about content from a single json file to separate HTML files which will make maintenance MUCH easier.
* Add support for using images in the `public` folder in the about widget content.

Waiting on feedback from WFRC before merging this...

Closes #69

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,59 @@
...
+## [3.0.0-3](https://github.com/agrc/wfrc/compare/v3.0.0-2...v3.0.0-3) (2020-10-13)
+
+
+### Bug Fixes
+
+* a little more breathing room for small text ([d916005](https://github.com/agrc/wfrc/commit/d916005df1857232cacc7b65537de9141acb0980))
+* add App component css to help replicate ([0eef309](https://github.com/agrc/wfrc/commit/0eef30998488b15c80d2d7a4cefa6b9d8e831f82))
+* align text of benefits sub-items ([6592be8](https://github.com/agrc/wfrc/commit/6592be8085004352867d7681e81eeef77d3f3133))
+* consistent top margin for section headers ([cfd7bb6](https://github.com/agrc/wfrc/commit/cfd7bb636a3d82a66fa48c2f00167b3115248a87))
+* replace vision with new content and remove temporary container ([11c8278](https://github.com/agrc/wfrc/commit/11c827858fb9e07819b37785ae583d23678f266e))
```
Comment (steveoh) [inline CHANGELOG.md:21]: **thought**: i wonder if these css and markup commits should use the style convention since they aren't really bug fixes.

**Review by steveoh — APPROVED** [review]
great job with the conversion!

File: CHANGELOG.md (hunk 1 above)
Comment (stdavis) [reply CHANGELOG.md:21]: Hmm. I haven't really used that tag before. I guess was thinking it was for code-style.

Comment (steveoh) [reply CHANGELOG.md:21]: yeah the angular doc does define style that way so maybe that wasn't the best suggestion. The take away should be that anything using feat or fix will end up in the customer facing change log. I'm not sure these items are then helpful there.

**Pull request merged**